packages/propel/src/emoji-icon-picker/lucide-icons.tsx

```
@@ -130,7 +134,7 @@ import {
   LogIn,
   LogOut,
   Mail,
-  Map as MapIcon,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)